### ISSUE TITLE
[ComboBox]: Missing Focus Outline

### ISSUE THREAD

**Issue by Martin-Rippel** [UI5/webcomponents#13444]
### Bug Description

Steps to reproduce:

- Open [ComboBox | UI5 Web Components](https://ui5.github.io/webcomponents/nightly/components/ComboBox/#basic-sample)
- Open the dropdown and start clicking a list item
- Actual: No focus outline appears for the clicked list item
- Expected: A focus outline appears for the clicked list item

### Steps to Reproduce

1.
2.
3.
...

### Priority

None

### UI5 Web Components Version

2.22.0-rc.3

### Browser

Edge

### Declaration

- [x] I’m not disclosing any internal or sensitive information.
